tests/test_help.py

```diff
@@ -19,3 +19,22 @@ def test_system_ssl_py26():
 def test_system_ssl():
     """Verify we're actually setting system_ssl when it should be available."""
     assert info()['system_ssl']['version'] != ''
+
+
+class VersionedPackage(object):
+    def __init__(self, version):
+        self.__version__ = version
+
+
+def test_idna_without_version_attribute(mocker):
+    """Older versions of IDNA don't provide a __version_ attribute, verify
```

Comment:
Sorry, one final note and I think we're set here. Could we get the second underscore for the closing dunder on version (`__version_` -> `__version__`). Once that's done I think I'm happy with this.